interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html

```diff
@@ -2,7 +2,6 @@
 [-include file="$TPL_NAME/views/header.html"-]
 
 [-* we want to include our stylesheet for this view*-]
-<link rel="stylesheet" href="[-$TPL_STYLE_PATH-]/ajax_search.css" type="text/css">
```

Comment:
fare thee well interface/main/calendar/modules/PostCalendar/pntemplates/default/style/ajax_search.css, you will NOT be missed :)